README.md

```diff
@@ -20,7 +20,14 @@ Run:
     ```yarn test```
 
 
-## Instructions
+## Overview
 
-Please complete the exercises provided on the 'Question One', 'Question Two' and 'Question Three' pages
+Hi! Welcome to the Skedulo frontend tech project. You will see we have 3 questions and each of their instructions are located in the local INSTRUCTIONS.md. Please complete these exercises then save to a private github repository. Once you're ready reach out to your hiring manager and they will share some github names to add as collaborators who will review your submission.
```

Comment:
I can see that in the old PRs, there were some candidates who created PR to submit their answers to this repo. Maybe, we can also add a sentence to clarify that they don't need to create a PR for their answers?

Reply:
There are a few but PRs aren't welcome on this repo from candidates as we want to ensure they submit through a private repo. Trying to ensure that other candidates don't steal the work of previous candidates

Reply:
Yep, it's important to ensure that the candidate doesn't cheat while doing our test.